web-animations-2/Overview.bs

```diff
@@ -1229,6 +1240,42 @@ Update the description of an effect that targets a non-animated property as:
 >     delaying the fulfilment of an <a>animation</a>'s <a>current finished
 >     promise</a>.
 
+<h3 id="keyframe-effects">Keyframe Effects</h3>
+
+<h4 id="the-effect-value-of-a-keyframe-animation-effect">The effect value of a keyframe effect</h4>
+
+The procedure for computing the <a>effect value</a> of a single property
+referenced by a <a>keyframe effect</a> as one of its <a>target properties</a>,
+for a given <var ignore=''>iteration progress</var>, <var>current iteration</var> and
+<var ignore=''>underlying value</var> is amended by inserting the following step:
```

Comment:
...is amended by inserting the following step after the step to apply the keyframe effect composite mode.

Reply:
Done.